impeller/renderer/backend/gles/blit_command_gles.cc

```diff
@@ -66,16 +66,78 @@ std::string BlitCopyTextureToTextureCommandGLES::GetLabel() const {
   return label;
 }
 
+namespace {
+static GLfloat pixels_to_gl_coords(GLfloat position, GLfloat pixels) {
```

Comment:
This doesn't match the style guides naming structure: 
https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Done